libMBIN/Source/NMS/GameComponents/GcCreatureComponentData.cs

```diff
@@ -1,10 +1,11 @@
-﻿using libMBIN.NMS.Toolkit;
+﻿using System.Collections.Generic;
+
+using libMBIN.NMS.Toolkit;
 using libMBIN.NMS.GameComponents;
-using System.Collections.Generic;
 
 namespace libMBIN.NMS.GameComponents
 {
-	[NMS(Size = 0x50, Alignment = 0x10, GUID = 0x5184B44469E17BE1, NameHash = 0x28E8DAB19EC55198)]
+    [NMS(Size = 0x50, GUID = 0xCF20BAC0F40D296B, NameHash = 0x28E8DAB19EC55198)]
```

Comment:
I'd probably keep this alignment to be safe...

Reply:
Done